### DISCUSSION TITLE
Evaluation Queue Stuck

### DISCUSSION THREAD

**Discussion by christopherpeers** [cyberbotics/wrestling#50] [General]
Hello,

I don't think anything is going through the evaluation queue right now as it's somehow stuck.

Thanks.

**Comment by omichel** [comment]
Yes... Let me fix this up...

**Comment by omichel** [comment]
It should be fixed now. Sorry for the trouble.

**Comment by omichel** [comment]
By the way, very nice goblin mode! 🤣
